### PR TITLE
fix: catch error on loadPost when watch fs change

### PR DESCRIPTION
**Pull request by wangbinyq**
If there were errors when parse markdown file on loadPost it will break watchFs functional.

### CODE REVIEW

File: blog.tsx [hunk 1]
```diff
@@ -204,9 +204,12 @@ async function watchForChanges(postsDirectory: string) {
     if (event.kind === "modify" || event.kind === "create") {
       for (const path of event.paths) {
         if (path.endsWith(".md")) {
-          await loadPost(postsDirectory, path);
-          HMR_SOCKETS.forEach((socket) => {
-            socket.send("refresh");
+          await loadPost(postsDirectory, path).then(() => {
```
Comment (ije) [inline blog.tsx:207]: can you please use `try catch` ?

Comment (wangbinyq) [reply blog.tsx]: updated

Comment (ije) [reply blog.tsx]: thanks

**Review by ije — APPROVED** [review]
LGTM

**Pull request merged**